apps/crn-frontend/src/network/teams/TeamProfile.tsx

```diff
@@ -71,15 +74,17 @@ const TeamProfile: FC<TeamProfileProps> = ({ currentTime }) => {
   const [teamListElementId] = useState(`team-list-${uuid()}`);
   const { teamId } = useRouteParams(route);
   const team = useTeamById(teamId);
+  const user = useCurrentUserCRN();
+  const isStaff = user?.role === 'Staff';
 
   useEffect(() => {
     // eslint-disable-next-line @typescript-eslint/no-floating-promises
     loadAbout()
-      .then(team?.tools ? loadWorkspace : undefined)
+      .then(team?.tools || isStaff ? loadWorkspace : undefined)
```

Comment:
Is this right? In the backend it seems we show tools to everyone that belongs to the team https://github.com/yldio/asap-hub/blob/master/apps/crn-server/src/routes/team.route.ts#L39-L41

If that has changed for sure, I think it would be nice to keep the rule in the backend as it was before :)

Reply:
oh sorry, I don't know why I skipped the `||` in the change above 😅 
But I still think it would be nice to have all rules in the same place (here https://github.com/yldio/asap-hub/blob/master/apps/crn-server/src/routes/team.route.ts#L39-L41)